doc/manual/modules.rst

```diff
@@ -257,24 +257,32 @@ the statements in a module often involves compiling a large amount of code.
 Julia provides the ability to create precompiled versions of modules
 to reduce this time.
 
-There are two mechanisms that can achieve this:
-incremental compile and custom system image.
-
-To create a custom system image that can be used when starting Julia with the ``-J`` option,
-recompile Julia after modifying the file ``base/userimg.jl`` to require the desired modules.
```

Comment:
It doesn't have anything to do with Modules, so it doesn't belong in the module docs, and it's already described in more useful detail in the devdocs where it belongs.